internal/features/rootmodules/events.go

```diff
@@ -246,16 +241,7 @@ func (f *RootModulesFeature) indexTerraformSourcesDirs(ctx context.Context, dir
 
 	for _, subDir := range f.Store.TerraformSourcesDirectories(dir.Path()) {
 		dh := document.DirHandleFromPath(filepath.Join(dir.Path(), subDir))
-
-		// notify the event bus that a module has been opened
-		// to trigger decoding of the module and its providers
-		// this is done differently for declared module calls in normal TF code
-		spawnedIds := f.eventbus.DidOpen(eventbus.DidOpenEvent{
-			Context:    ctx,
-			Dir:        dh,
-			LanguageID: lsp.Terraform.String(),
-		})
-		jobIds = append(jobIds, spawnedIds...)
+		f.stateStore.WalkerPaths.EnqueueDir(ctx, dh)
```

Comment:
This doesn't trigger the event from the root modules feature anymore, however ... (see further down below)